persistence/src/main/scala/hmda/persistence/demo/DemoInstitutions.scala

```diff
@@ -46,7 +46,10 @@ object DemoInstitutions extends ResourceUtils {
   }
 
   def toStatus(text: String): InstitutionStatus = {
-    InstitutionStatus.values.filter(status => status.entryName == text).head
+    text match {
+      case "active" => Active
```

Comment:
I know we're matching on specific strings here... My reasoning is that this file's purpose is to parse `persistence/src/main/resources/demoInstitutions.csv`, which uses its own system for representing institutions, not necessarily the strings that match `InstitutionStatusMessage`. 
But I could be convinced to change these to `activeMsg` and `inactiveMsg` if someone feels strongly.